entrypoint.sh

```diff
@@ -17,13 +17,32 @@ _RUNNER_NAME=${RUNNER_NAME:-${RUNNER_NAME_PREFIX:-github-runner}-$(head /dev/ura
 _RUNNER_WORKDIR=${RUNNER_WORKDIR:-/_work}
 _LABELS=${LABELS:-default}
 _RUNNER_GROUP=${RUNNER_GROUP:-Default}
-_SHORT_URL=${REPO_URL}
 _GITHUB_HOST=${GITHUB_HOST:="github.com"}
 
-if [[ ${ORG_RUNNER} == "true" ]]; then
-  _SHORT_URL="https://${_GITHUB_HOST}/${ORG_NAME}"
+# ensure backwards compatibility
+if [[ -z $RUNNER_SCOPE ]]; then
+  if [[ ${ORG_RUNNER} == "true" ]]; then
+    export RUNNER_SCOPE="org"
+  else
+    export RUNNER_SCOPE="repo"
+  fi
 fi
 
+case ${RUNNER_SCOPE} in
+  "org")
```

Comment:
for string based we should make it case insensitive and allow shortening for things like org|organization